examples/helpers/tiled.rs

```diff
@@ -365,6 +369,9 @@ pub fn process_loaded_maps(
                                     })
                                     .id();
                                 tile_storage.set(&tile_pos, tile_entity);
+                                // Make tiles transform relative to layer transform, 
+                                // to ensure they have correct world_pos aligned with drawn location
+                                commands.entity(layer_entity).add_child(tile_entity);
```

Comment:
Tiles don't have bevy transforms. This wont do what you want it to here. Due to how bevy's transform propagation works it would be incredible slow to have each tile have a bevy transform so I can't recommend that.